tests/testdata/models/run_demo_model.json

```diff
@@ -10,100 +10,45 @@
     "assets": {
         "0": {
             "name": "OS App",
-            "type": "Application"
+            "type": "Application",
+            "associated_assets": {}
```

Comment:
None of these assets have their associations left anymore. Why do we want this model then?